llvm/test/Instrumentation/NumericalStabilitySanitizer/cfg.ll

```diff
@@ -0,0 +1,113 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt -mtriple=x86_64-linux-gnu -passes=nsan -nsan-shadow-type-mapping=dqq -S %s | FileCheck %s
```

Comment:
This should go in test/Instrumentation/NumericalStabilitySanitizer/X86, or needs a REQUIRES: x86-registered-target

Reply:
Removed -mtriple=x86_64-linux-gnu